Frontend/lib/data/api/api_handlers/cosign_api.dart

```diff
@@ -21,7 +21,7 @@ class CoSignApi {
           queryParameters: queryParameters,
           options: token == null
               ? null
-              : Options(headers: {'Authorization': 'Bearer $token'}));
+              : Options(headers: {'Authorization': 'Token $token'}));
 
       return ApiUtils.toApiResponse(response);
     } on DioError catch (e) {
```

Comment:
Just curious, what is the meaning of DioError?

Reply:
Dio is the name of the package and a powerful Http client for Dart.